metaconfig/src/main/scala/org/scalafmt/util/BuildTime.scala

```diff
@@ -0,0 +1,18 @@
+package org.scalafmt.util
+
+import scala.annotation.compileTimeOnly
+import scala.meta._
+
+@compileTimeOnly("Gets expanded away")
```

Comment:
This message appears when users forget to add the paradise compiler plugin. A clearer error might be:

"@BuildTime not expanded. Have you enabled the scala.meta paradise plugin?" 

Same for GitCommit.